Libraries/Text/Text/RCTTextView.m

```diff
@@ -149,12 +206,34 @@ - (void)setTextStorage:(NSTextStorage *)textStorage
           contentFrame:(CGRect)contentFrame
        descendantViews:(NSArray<RCTUIView *> *)descendantViews // TODO(macOS ISS#3536887)
 {
+  _contentFrame = contentFrame;
+
 #if TARGET_OS_OSX // [TODO(macOS GH#774)
-  [self removeAllTextStorageLayoutManagers];
+  _textStorage = textStorage;
 #endif // ]TODO(macOS GH#774)
 
-  _textStorage = textStorage;
-  _contentFrame = contentFrame;
+#if TARGET_OS_OSX // [TODO(macOS GH#774)
```

Comment:
should we merge this with previous `#if TARGET_OS_OSX`?

Reply:
We can't.
This is actually a bug as we learned this week > D39356602